### PR TITLE
feat: adding list component

### PR DESCRIPTION
**Pull request by mariyasada**
adding a list to component library and changing carbon iframe link to thiscodeworks link. do it review

### CODE REVIEW

**Comment by Manju-sree** [comment]
Code looks fine

**Review by Manju-sree — APPROVED** [review]
Code looks fine , you can merge

File: htmlFiles/avatar.html [hunk 1]
```diff
@@ -160,9 +160,23 @@ <h2 class="section-heading section-code-example-container">
                     </h2>
 
                     <div class="code-snippet" id="avatar-div-code"> 
-                    <img class="code-example" src="/images/avatarImage/code.png"/>
+                    <!-- <img class="code-example" src="/images/avatarImage/code.png"/>
```
Comment (t007rushi) [inline htmlFiles/avatar.html:163]: You can remove unwanted comments here

File: htmlFiles/badges.html [hunk 2]
```diff
@@ -202,9 +202,23 @@ <h2 class="section-heading section-code-example-container">
                 </h2>
 
                 <div class="code-snippet" id="badges-div-code"> 
-                        <img class="code-example" src="/images/badgeImage/code.png"/>
+                        <!-- <img class="code-example" src="/images/badgeImage/code.png"/>
```
Comment (t007rushi) [inline htmlFiles/badges.html:205]: You can remove unwanted comments

File: stylesheets/card/card.css [hunk 3]
```diff
@@ -133,8 +132,23 @@ button
...
+}
+.horizontal-card .card-image
+{
+    height:100%;
+    min-width:14rem;
+}
+.button-container
+{
+    display: flex;
+    flex-direction: row;
```
Comment (t007rushi) [inline stylesheets/card/card.css:150]: no need to write default flex direction + as root is set 62.5 box-size no need of explicit 1rem

**Review by t007rushi — COMMENTED** [review]
You can do those changes to improve code

File: htmlFiles/avatar.html (hunk 1 above)
Comment (mariyasada) [reply htmlFiles/avatar.html:163]: okay i will do that

**Pull request merged**